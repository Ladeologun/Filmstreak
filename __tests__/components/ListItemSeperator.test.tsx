import renderer from 'react-test-renderer';
import {test,expect} from '@jest/globals';
import { ListItemSeperator } from "~components";


test('renders correctly', () => {
    const tree = renderer.create(<ListItemSeperator />).toJSON();
    expect(tree).toMatchSnapshot();
});
