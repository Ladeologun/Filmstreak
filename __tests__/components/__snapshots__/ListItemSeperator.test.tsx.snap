// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#0F111D",
      "height": "100%",
      "width": 10,
    }
  }
/>
`;
